main/src/vtr_navigation/include/vtr_navigation/graph_map_server.hpp

```diff
@@ -189,6 +201,12 @@ class GraphMapServer : public tactic::Graph::Callback,
   rclcpp::Subscription<MoveGraphMsg>::SharedPtr move_graph_sub_;
   rclcpp::Subscription<AnnotateRouteMsg>::SharedPtr annotate_route_sub_;
   rclcpp::Subscription<UpdateWaypointMsg>::SharedPtr update_waypoint_sub_;
+
+  rclcpp::Subscription<NavSatFix>::SharedPtr pose_pub_;
+
+  const float dist_thres_ = 0.1;
+  bool initial_pose_set_ = false;
+  boost::circular_buffer<std::pair<int, int>> gps_coords_ {2};
```

Comment:
Why do we need to use a boost data structure? Is there no standard library one that would work?